### PR TITLE
[Kl-547] UI req simplify accounts user flow

### PR DESCRIPTION
**Pull request by thesvyatoslavtt**

### CODE REVIEW

File: libs/ui/src/components/navigator/hooks/use-navigation-from-sensetive-screens.hook.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { ScreensEnum } from '../../../enums/sreens.enum';
+import { useNavigation } from '../../../hooks/use-navigation.hook';
+
+const sensetiveRoutes = [ScreensEnum.RevealPrivateKey, ScreensEnum.RevealSeedPhrase];
```
Comment (Boltman92) [inline libs/ui/src/components/navigator/hooks/use-navigation-from-sensetive-screens.hook.ts]: sensitive is correct spelling, please change

File: libs/ui/src/components/screen-components/header-container/components/screen-title/screen-title.tsx [hunk 2]
```diff
@@ -27,9 +27,9 @@ export const ScreenTitle: FC<Props> = ({
 }) => (
   <Column style={styles.root}>
     {isDefined(onBackButtonPress) && (
-      <TouchableIcon name={IconNameEnum.ArrowLeft} onPress={onBackButtonPress} style={styles.icon} testID={testID} />
+      <TouchableIcon name={IconNameEnum.ArrowLeft} onPress={onBackButtonPress} style={styles.icon} />
     )}
-    <Text style={[styles.title, titleStyle]} numberOfLines={numberOfLines} ellipsizeMode="tail">
+    <Text style={[styles.title, titleStyle]} numberOfLines={numberOfLines} ellipsizeMode="tail" testID={testID}>
```
Comment (Boltman92) [inline libs/ui/src/components/screen-components/header-container/components/screen-title/screen-title.tsx]: ellipsizeMode is tail by default, could be removed

File: libs/ui/src/enums/sreens.enum.ts [hunk 3]
```diff
@@ -67,9 +68,13 @@ export enum ScreensEnum {
   Swap = 'Swap',
   SwapRoute = 'SwapRoute',
   SlippageTolerance = 'SlippageTolerance',
-  EditPermission = 'EditPermission'
+  SwapConfirmation = 'SwapConfirmation',
+  EditPermission = 'EditPermission',
+  ConfirmAccess = 'ConfirmAccess'
 }
 
+type SensetiveScreensEnum = ScreensEnum.RevealPrivateKey | ScreensEnum.RevealSeedPhrase;
```
Comment (Boltman92) [inline libs/ui/src/enums/sreens.enum.ts]: also sensItive

File: libs/ui/src/screens/settings/screens/accounts-settings/hooks/use-accounts-transformer.hook.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import { useMemo } from 'react';
+
+import { AccountTypeEnum } from '../../../../../enums/account-type.enum';
+import { useFilteredAccounts } from '../../../../../hooks/use-filtered-accounts.hook';
+import { useAllAccountsSelector } from '../../../../../store/wallet/wallet.selectors';
+import { AccountsTransformer } from '../interfaces/accounts.interface';
+
+export const useAccountsTransformer = () => {
```
Comment (Boltman92) [inline libs/ui/src/screens/settings/screens/accounts-settings/hooks/use-accounts-transformer.hook.ts:8]: this hook looks like overperform. 
Why we just can't fitler accounts by accounts type?

**Review by Boltman92 — DISMISSED** [review]

**Review by Boltman92 — DISMISSED** [review]

**Review by yarmoshevych — DISMISSED** [review]

**Pull request merged**